log/log.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+)
+
+var logger zerolog.Logger
```

Comment:
MM uses a nice logger may we may just re-use this one or use `logrus` even better. Logrus is the most popular library out there for Golang

Reply:
Palantirs go github app library uses zerolog so we used that one not to have two different logger systems.

https://github.com/palantir/go-githubapp#structured-logging

Reply:
hmm I see.let's keep it for now